src/Centreon/Domain/HostConfiguration/HostMacro.php

```diff
@@ -174,4 +179,20 @@ public function setOrder(?int $order): HostMacro
         $this->order = $order;
         return $this;
     }
+
+    /**
+     * @return int|null
+     */
+    public function getHostId(): ?int
+    {
+        return $this->hostId;
+    }
+
+    /**
+     * @param int|null $hostId
+     */
+    public function setHostId(?int $hostId): void
+    {
+        $this->hostId = $hostId;
+    }
```

Comment:
service macro entity returns $this.
To be consistent, it should be done in the same way for host macro entity